src/components/DataManager/Toolbar/ActionsButton.js

```diff
@@ -97,45 +98,56 @@ export const ActionsButton = injector(observer(({ store, size, hasSelected, ...r
       </Block>
     );
 
-    return isFFLOPSE3 ? (
-      hasChildren ? (
+    //&& action.newStyle
+    return hasChildren  ? (
+      <Dropdown.Trigger 
+        key={action.id}
+        align="top-right-outside"
+        toggle={false}
+        ref={submenuRef}
+        content={<Block name='actionButton-submenu' tag="ul" mod={{ newUI: isNewUI }}>{action.children.map(ActionButton, parentRef)}</Block>}
+      >
+        {titleContainer}
+      </Dropdown.Trigger>  
+    ) : (
+      isNewUI ? (
         <Dropdown.Trigger 
           key={action.id}
           align="top-right-outside"
           toggle={false}
           ref={submenuRef}
-          content={<Block name='actionButton-submenu' tag="ul">{action.children.map(ActionButton, parentRef)}</Block>}
+          content={<Block name='actionButton-submenu' tag="ul" mod={{ newUI: isNewUI }}>{action.children.map(ActionButton, parentRef)}</Block>}
         >
           {titleContainer}
         </Dropdown.Trigger>
-      ) : titleContainer
-    ) : (
-      <Menu.Item
-        size={size}
-        key={action.id}
-        danger={isDeleteAction}
-        onClick={() => {
-          invokeAction(action, isDeleteAction);
-        }}
-        icon={isDeleteAction && <FaTrash />}
-      >
-        {action.title}
-      </Menu.Item>
+      ) : (
+        <Menu.Item
+          size={size}
+          key={action.id}
+          danger={isDeleteAction}
+          onClick={() => {
+            invokeAction(action, isDeleteAction);
+          }}
+          className={`actionButton${action.isSeperator ? "_isSeperator" : (action.isTitle ? "_isTitle" : "")}`}
```

Comment:
typo in the word `isSeperator` -> `isSeparator`

Reply:
woops that type-o is likely every where lol i'll fix it